app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -8,10 +8,19 @@
 
 <% if @presenter.otp_expiration.present? %>
   <%= render CountdownAlertComponent.new(
-        show_at_remaining: IdentityConfig.store.otp_expiration_warning_seconds.seconds,
-        alert_options: { class: 'margin-bottom-4' },
-        countdown_options: { expiration: @presenter.otp_expiration },
+      show_at_remaining: IdentityConfig.store.otp_expiration_warning_seconds.seconds,
+      alert_options: { class: 'margin-bottom-4' },
+      countdown_options: { expiration: @presenter.otp_expiration },
+    ) %>
+
+
+    <p class="usa-sr-only" role="timer" aria-live="polite" aria-atomic="true">
+      <%= render CountdownComponent.new(
+        expiration: Time.zone.now,
+        update_interval: 30.seconds,
+        start_immediately: true,
```

Comment:
One thing we probably want to be conscious of here is that I don't know that we want the timer to start announcing "9 minutes and 30 seconds remaining" to a screen reader user, and for every 30 seconds thereafter. The alert component above is configured to only start counting down once it gets closer to the timeout (2m30s IIRC).

Reply:
Yeah the tricky thing that I _think_ is happening is that we are dealing with two timers that need to tick at 2 different paces. Then when the banner shows up, it is counting down by the `update_interval` of one second while the countdown changes every 5 seconds. There has to be a way in which we silence the alert so that we only hear from the countdown, but I'm thinking of how to do that right now.

This feature works differently from the session timer because the popup shows once and the times are synced together.